spec/features/peep_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../../app/models/peep'
+
+feature "Peep Management" do
+
+  let(:dan) do
+    User.create(name: "dan",
+                email: "test@test.com",
+                username: 'super_tester',
+                password: 'test')
+  end
+
+  let(:sam) do
+    User.create(name: "sam",
```

Comment:
Don't like Class in Unit Test other than tested Class -> Use FactoryGirl to make mocks for you